wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -8,9 +8,9 @@
     <main class="content-wrapper w-overflow-x-hidden" id="main">
         <div class="content">
             {# Always show messages div so it can be appended to by JS #}
-            <div class="messages" role="status">
-                {% if messages %}
-                    <ul>
+            <div class="messages" role="status" data-controller="w-messages" data-action="w-messages:add@document->w-messages#add" data-w-messages-added-class="new" data-w-messages-show-class="appear">
+                <ul data-w-messages-target="messages">
```

Comment:
```suggestion
                <ul data-w-messages-target="container">
```

let's rename this container, we are using this naming for the 'target thing to add stuff to' in other planned controllers. Avoids this being too tightly coupled to this one use case.